### ISSUE TITLE
Mobile Data NOT working

### ISSUE THREAD

**Issue by Umar-oss** [GrapheneOS/os-issue-tracker#4488]
After the last release,  when I turn mobile Data on I get no internet

**Comment by Umar-oss** [comment]
Pixel 6a 
It just happened after I installed the update

**Comment by WaleedMortaja** [comment]
I do not think this is a GrapheneOS issue. I have the same phone, updated to the latest release. Mobile Data works fine.
